README.md

```diff
@@ -104,13 +104,13 @@ When running cmake, next options can be used to change the output libraries/Pal/
 - `BUILD_CURL_TRANSPORT`: This option would build an HTTP transport library using CURL. It requires libcurl to be installed (vcpkg or globally). This option will make samples to be linked with this HTTP and be functional to send HTTP requests<br>
 use it as
 ```bash
-cmake -DBUILD_CURL_TRANSPORT ..
+cmake -DBUILD_CURL_TRANSPORT=ON ..

```

Comment:
Why are these steps repeated in two places. I think we should just have them in one place (contributing guide) and link to that in the README. Otherwise, we will keep having the out-of-sync issue.

Reply:
yeah, I don't really this this either.. 
I think I will make Contribute.md to reference the Getting Started from README for learning about how to build.
Then in contribute, we should probably only point to out guidelines docs.
I will push a new commit for this